cdap-ui/app/features/home/unauthorized-namespace.html

```diff
@@ -0,0 +1,20 @@
+<!--
+  Copyright © 2016 Cask Data, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+
+<div class="jumbotron text-center">
+  <h1><span class="text-danger">uh oh!</span></h1>
+  <p>You do not have authorization to view any namespaces. Please contact your system administrator to get access to a namespace.</p>
```

Comment:
Isn't it more like - unauthorized to view a single/particular namespace?
This message says not authorized to view *any* namespaces.